client/src/lib/votecounting.ts

```diff
@@ -550,6 +570,7 @@ const processCastVoteRecord = ({
     _testBallot: castVoteRecord._testBallot,
     _scannerId: castVoteRecord._scannerId,
     _pageNumber: castVoteRecord._pageNumber,
+    _pageNumbers: castVoteRecord._pageNumbers,
```

Comment:
Should we be normalizing them into one field? Or keep both?

Reply:
I think we keep both, good to capture that it is old vs new format.